.pre-commit-config.yaml

```diff
@@ -0,0 +1,6 @@
+repos:
+-   repo: https://github.com/spinoch/blackadder
+    rev: master
+    hooks:
+    - id: blackadder
+      language_version: python3
```

Comment:
Same comment w/ precommit (I wouldn't mind exploring adding this in another PR though)